src/web/demo.rkt

```diff
@@ -240,143 +240,94 @@
-             (hash-set! *completed-jobs* hash (hasheq 'points exacts))
-             (eprintf " complete\n")
-             (hash-remove! *jobs* hash)
-             (semaphore-post sema)]
-            [(list 'evaluate hash formula sema seed sample)
-             (define test (parse-test formula))
-             (define pcontext (json->pcontext sample (test-context test)))
-             (eprintf "Evaluation job started on ~a..." formula)
-             (define result (run-herbie 'evaluate test #:seed seed 
-              #:pcontext pcontext #:profile? #f #:timeline-disabled? #t))
-             (define approx (job-result-backend result))
-             (hash-set! *completed-jobs* hash (hasheq 'points approx))
-             (eprintf " complete\n")
-             (hash-remove! *jobs* hash)
-             (semaphore-post sema)]
-            [(list 'cost hash formula sema)
-             (define test (parse-test formula))
-             (eprintf "Computing cost of ~a..." formula)
-             (define result (run-herbie 'cost test 
-              #:profile? #f #:timeline-disabled? #t))
-             (define cost (job-result-backend result))
-             (hash-set! *completed-jobs* hash (hasheq 'cost cost))
-             (eprintf " complete\n")
-             (hash-remove! *jobs* hash)
-             (semaphore-post sema)])
+         [(list 'sample hash formula sema args)
+          (define seed (first args))
+          (define test (parse-test formula))
+          (eprintf "Sampling job started on ~a..." formula)
+          (define result (run-herbie 'sample test #:seed seed 
+          #:profile? #f #:timeline-disabled? #t))
+          (hash-set! *completed-jobs* hash result)
+          (eprintf " complete\n")
+          (hash-remove! *jobs* hash)
+          (semaphore-post sema)]
+         [(list 'errors hash formula sema args)
+          (define seed (first args))
+          (define sample (second args))
+          (define test (parse-test formula))
+          (define pcontext (json->pcontext sample (test-context test)))
+          (eprintf "Analyze job started on ~a..." formula)
+          (define result (run-herbie 'errors test
+          #:seed seed #:pcontext pcontext
+          #:profile? #f #:timeline-disabled? #t))
+          (hash-set! *completed-jobs* hash result)
+          (eprintf " complete\n")
+          (hash-remove! *jobs* hash)
+          (semaphore-post sema)]
+         [(list 'local-error hash formula sema args)
+          (define seed (first args))
+          (define sample (second args))
+          (define test (parse-test formula))
+          (define pcontext (json->pcontext sample (test-context test)))
+          (eprintf "Local error job started on ~a..." formula)
+          (define result (run-herbie 'local-error test #:seed seed 
+          #:pcontext pcontext #:profile? #f #:timeline-disabled? #t))
+          (hash-set! *completed-jobs* hash result)
+          (eprintf " complete\n")
```

Comment:
I noticed that `'improve` jobs print log lines in a different format from all others. Why?